test/functional/feature_miniscript.py

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 James Chiang
```

Comment:
Please change this to "The Bitcoin Core developers" or add your name on line 105 in https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/copyright_header.py

Otherwise it will get overridden by the copyright-headers script

Reply:
Thank you for the review and tip, will gladly amend copyright header as you suggested!